src/vl-rich-data-field.js

```diff
@@ -164,15 +167,21 @@ export class VlRichDataField extends vlElement(HTMLElement) {
     }));
   }
 
+  get __headerContent() {
+    return this.label || (this._labelSlotElement ? this._labelSlotElement.innerHTML : undefined);
+  }
+
   __getHeaderContentElement() {
-    const text = this.label || `${this._labelSlotElement.innerHTML}`;
-    if (this.sortable) {
-      const direction = this.sortingDirection ? `data-vl-direction="${this.sortingDirection}"` : '';
-      const priority = this.sortingPriority ? `data-vl-priority="${this.sortingPriority}"` : '';
-      const sorter = `<vl-rich-data-sorter data-vl-for="${this.name}" ${direction} ${priority}></vl-rich-data-sorter>`;
-      return this._template(`<a>${text}${sorter}</a>`);
-    } else {
-      return this._template(`${text}`);
+    const content = this.__headerContent;
+    if (content) {
+      if (this.sortable) {
+        const direction = this.sortingDirection ? `data-vl-direction="${this.sortingDirection}"` : '';
+        const priority = this.sortingPriority ? `data-vl-priority="${this.sortingPriority}"` : '';
+        const sorter = `<vl-rich-data-sorter data-vl-for="${this.name}" ${direction} ${priority}></vl-rich-data-sorter>`;
+        return this._template(`<a>${content}${sorter}</a>`);
+      } else {
+        return this._template(`${content}`);
+      }
     }
```

Comment:
als t een functie is, dus een methode die iets returnt, zie ik graag op alle paden ne return staan, dus ook nen else (maar dat is misschien voor een stuk persoonlijke smaak)

Reply:
Ikzelf vind het prima om in JS te steunen op de impliciete `return undefined`, maar ik snap dat je het liever expliciet wilt zien staan.